README.md

```diff
@@ -1,15 +1,72 @@
 eslint-config-simplabs
 ==============================================================================
 
+[![Latest NPM release][npm-badge]][npm-badge-url]
+[![TravisCI Build Status][travis-badge]][travis-badge-url]
+
+[npm-badge]: https://img.shields.io/npm/v/eslint-config-simplabs.svg
+[npm-badge-url]: https://www.npmjs.com/package/eslint-config-simplabs
+[travis-badge]: https://img.shields.io/travis/simplabs/eslint-config-simplabs/master.svg?label=TravisCI
+[travis-badge-url]: https://travis-ci.org/simplabs/eslint-config-simplabs
+
 Shareable ESLint config for all [simplabs](https://simplabs.com/) projects
 
+
 Installation
 ------------------------------------------------------------------------------
 
 ```
 ember install eslint-config-simplabs
 ```
 
+or if you don't used Ember CLI in your project:
```

Comment:
This sounds like you're referring to legacy non-Ember CLI Ember projects while it actually just refers to non-Ember projects (unless I'm mistaken).